### DISCUSSION TITLE
esp32-s3-usb-otg with st7789 1.3" display, call me IMPRESSED!

### DISCUSSION THREAD

**Discussion by HeadHodge** [Bodmer/TFT_eSPI#2577] [Compatible displays and setup files]
After changing setup settings, examples worked out of the box! Easier to use than Espressif LCD Library. 😎👍

Settings used:

// For ESP32-S3-USB-OTG Dev board (only tested with st7789 1.3" display)
#define ST7789_DRIVER                     // Full configuration option, define additional parm
#define TFT_DC   4                             // Data Command control pin - must use a pin
#define TFT_CS   5                              // Chip select control pin (library pulls permanently
#define TFT_SCLK 6                            //
#define TFT_MOSI 7                           //
#define TFT_RST  8                             // Reset pin, toggles on startup
#define TFT_BL   9                              // LED back-light control pin
#define TFT_BACKLIGHT_ON HIGH    // Level to turn ON back-light (HIGH or LOW)
#define SPI_FREQUENCY  20000000  //
#define TFT_WIDTH  240                   // ST7789 240 x 240 and 240 x 320
#define TFT_HEIGHT 240                   // ST7789 240 x 240
#define TFT_INVERSION_OFF            //
#define TFT_RGB_ORDER TFT_RGB    // Colour order Red-Green-Blue
`

**Comment by Bodmer** [comment]
Great to hear of your success. Enjoy!

**Comment by HeadHodge** [comment]
Hi,

Saw your resistor codes in your ".h" file, which brought nostalgic memories.

I was taught in school(air force):


   - *Big boys rape our young girls but Violet goes willingly.*



   - 0 = *B*lack
   - 1 = *B*rown
   - 2 = *R*ed
   - 3 = *O*range
   - 4 = *Y*ellow
   - 5 = *G*reen
   - 6 = *B*lue
   - 7 = *V*iolet
   - 8 = *G*ray
   - 9 = *W*hite.


You're missing 'Violet' 😲 🤓👍

Have a great day!

**Comment by Bodmer** [comment]
Ha! Yes, I annoy colleagues by calling it purple because the band does not look violet to my eyes! I don't use a mnemonic, I just learnt the code without. The are other politically correct mnemonics these days.
